mcase/runtest.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Description: Modal test code
+#   Author: Alois Mahdal <amahdal@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2020 Red Hat, Inc.
+#
+#   This program is free software: you can redistribute it and/or
+#   modify it under the terms of the GNU General Public License as
+#   published by the Free Software Foundation, either version 2 of
+#   the License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE.  See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program. If not, see http://www.gnu.org/licenses/.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include Beaker environment
+. /usr/bin/rhts-environment.sh || exit 1
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "rlImport ControlFlow/mcase"
+    rlPhaseEnd
+
+    rlPhaseStartTest
+        rlPass
+    rlPhaseEnd
```

Comment:
If the intention of this test is to check if the library can be rlImported, then this solution would be better.

```suggestion
    rlPhaseStartTest
        rlRun "rlImport ControlFlow/mcase"
    rlPhaseEnd
```

Reply:
AFAIK there's not even that.

runtest.sh was provided only because it was required by some tools / meta-data; for what i care it could be dropped :)

Reply:
if we drop it we should remove the `test` attribute from the _main.fmf_
Libraries does not need to have the bundled test, actually.

Reply:
File is removed and also the 'test' key from main.fmf